internal/c/parts/audio/miniaudio.h

```diff
@@ -61739,34 +61823,6 @@ MA_API ma_result ma_decoder_init_memory(const void* pData, size_t dataSize, cons
 #define MA_HAS_PATH_API
 #endif
 
-#if defined(MA_HAS_PATH_API)
```

Comment:
The rest of the changes looks good. But I am not sure if modifying miniaudio.h is a good idea. I have seen mackron pushing changes to the miniaudio dev branch. Some of these seem useful and important when they make it to release. If we want to use the newer versions when it is released, we would need to modify the header again to accommodate our changes. Which I think should be avoided. My recommendation would be to keep miniaudio.h intact and make all our changes outside of it. Looking at these changes, I am quite certain it is not too much we are changing... but still.

Reply:
Yeah I agree, I didn't really feel good about it at the time but we were using that internal stuff and I didn't want to completely rewrite it when it functions perfectly fine. I'll revert the changes to `miniaudio.h` and then just copy their code elsewhere (with new function names) for our use.